platform/model/main.graphql

```diff
@@ -1,5 +1,6 @@
 """
 @model
+@versioned
```

Comment:
This is not working because of https://github.com/aerogear/graphback/pull/1301/files#r434476629

/cc @ssd71

Reply:
Solved, it was rather a schema override problem.